packages/go/dawgs/drivers/neo4j/node.go

```diff
@@ -64,10 +64,33 @@ func NewNodeQuery(ctx context.Context, tx innerTransaction) graph.NodeQuery {
 }
 
 func (s *NodeQuery) run(statement string, parameters map[string]any) graph.Result {
-	return s.tx.Run(statement, parameters)
+	return s.tx.Raw(statement, parameters)
 }
 
-func (s *NodeQuery) Execute(delegate func(results graph.Result) error, finalCriteria ...graph.Criteria) error {
+func (s *NodeQuery) Exec(finalCriteria ...graph.Criteria) error {
```

Comment:
DRY: this can wrap `NodeQuery#Query` and just pass in a no-op delegate.

Reply:
Removed the definition from the interface as it was a stutter anyway while mapping out the interfaces for PG. Patched up